style.css

```diff
@@ -18,4 +18,42 @@
   --subtitle-font-size: 28px;
   --button-font-size: 14px;
   --description-font-size: 12px;
+
+  --rect-border-color:rgb(227,227,227);
+  --rect-border-radius:8px;
+  --rect-gap:12px;
+
+  --logo-small:20px;
+
+  --dropdown-box-shadow:0px 4px 12px rgba(15,15,15,0.08);
+  --dropdown-v2-image-height:72px;
+
+  --dropdown-v2-gap:24px;
+}
+
+
+.w-400{
+  width: 400px;
+}
+.w-300{
+  width: 300px;
+}
+.h-20{
+height:20px;
+}
+.px-12{
```

Comment:
Add all padding classes to the root and use it in the respective component/parent class.